roles/yambo/tasks/main.yml

```diff
@@ -11,10 +11,10 @@
     dest: "{{ globalconfig.vm_codes_folder }}"
     remote_src: True
 
-# Note: --with-p2y-version=qexsd is needed for QE 6.2
+# Note: use ./configure --with-p2y-version=qexsd for QE 6.2 or higher with new xml schema
```

Comment:
Sorry to keep asking about something so trivial, but I don't get it -
we *are* using QE 6.2.1, so why is the `--with-p2y-version=qexsd` no longer needed?

Reply:
I get it... we are using the old XML schema.